app/views/dashboards/_latest_upload.html.erb

```diff
@@ -1,4 +1,3 @@
-<!--Prod flag for bah-6852. Remove if/else statement when approved.-->
 <% if production? && CSV_TYPES_NO_PROD_NAMES.include?(upload.csv_type) %>
```

Comment:
Did you mean to just remove the comment here and leave the if/else statement?

Reply:
Yes, thought was to leave this functionality in place in case there is another CSV type that is added and doesn't need to show up in production environment